src/Core/Commander/Providers/WMTS_Provider.js

```diff
@@ -112,16 +111,42 @@ WMTS_Provider.prototype.url = function(coWMTS, layer) {
  * @param {type} coWMTS : coord WMTS
  * @returns {WMTS_Provider_L15.WMTS_Provider.prototype@pro;_IoDriver@call;read@call;then}
  */
-WMTS_Provider.prototype.getXbilTexture = function(tile, layer) {
-    var coWMTS = tile.tileCoord;
+WMTS_Provider.prototype.getXbilTexture = function(tile, layer, parameters) {
+    var cooWMTS = tile.matrixSet[layer.options.tileMatrixSet][0];
```

Comment:
When I delete a `addImageryLayer` (with `name: 'ORTHOIMAGERY.ORTHOPHOTOS.CRS84'` by example) in html file [(here)](https://github.com/iTowns/itowns2/blob/e615c05969b802ed88949d7cd1e93bf0b286b905/index.html#L174).
There's a error `tile.matrixSet[layer.options.tileMatrixSet]` is `undefined`.
There's not this error in master 94a98e455813fc36a878852d887067369782bb7c